mica-webapp/src/main/conf/application.yml

```diff
@@ -51,7 +51,6 @@ spring:
     data:
         mongodb:
             uri: mongodb://localhost:27017/mica
-    profiles: prod
```

Comment:
What happens if present in legacy application.yml?

Reply:
legacy will default to "prod" if they never set it else will use the profile_name that they defined.
profile_name "dev" seem to be related to logging.